.github/workflows/test.yaml

```diff
@@ -78,3 +78,9 @@ jobs:
         run: |
           cargo llvm-cov --lcov --all --all-features --all-targets > lcov.info
           bash <(curl -s https://codecov.io/bash) -C ${{github.sha}} -f lcov.info
+
+      - name: Upload coverage to coveralls
+        uses: coverallsapp/github-action@master
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          path-to-lcov: lcov.info
```

Comment:
I don't understand this step, what does it do that the bash script would not?

For the comparison, don't we need a record of the base branch (`master`) before we can have it working? I don't think we have one at the moment, we'd get one when merging the PR and running the action on pushing to `master`? Or did I get lost somewhere?

Reply:
Oh I hadn't realised that this was for a different tool